tutorials/dhcpcd-dedibox/index.mdx

```diff
@@ -16,28 +16,43 @@ dates:
 
 [dhcpcd](https://roy.marples.name/projects/dhcpcd/) is an open-source DHCP client, supporting both IPv4, and IPv6. The feature-rich software can be used to configure and automate network-related parameters of your computer using the [DHCP protocol](http://en.wikipedia.org/wiki/Dynamic_Host_Configuration_Protocol).
 
-In this tutorial, you will learn how to configure your IPv6 subnet with DHCPCD on Debian 10 or later on a Scaleway Dedibox dedicated server.
+<Message type="note">
+Roy Marples' dhcpcd (DHCP client daemon) is not the same as the discontinued Internet Systems Consortium's dhcpd (DHCP (server) daemon).
+</Message>
+
+In this tutorial, you will learn how to configure your IPv6 subnet with DHCPCD on a Scaleway Dedibox dedicated server.
 
 <Macro id="iam-requirements" />
 
 <Message type="requirement">
   - You have an account and are logged into the [Dedibox console](https://console.online.net)
-  - You have a [Dedibox&reg; dedicated server](https://www.scaleway.com/en/dedibox) running on Debian 10 or later.
+  - You have a [Dedibox&reg; dedicated server](https://www.scaleway.com/en/dedibox)
   - You have root access to the server
 </Message>
 
 
 ## Installing DHCPCD
 
 1. Change into the root account of your server by using `su` and the password of the `root` user.
-2. Update the `apt` package manager repositories, then upgrade the software already installed on the machine to the latest version available in the repositories: 
+2. 
```

Comment:
remove extra number here and in the lines that follow
```suggestion
```

Reply:
Ok